src/Value/Size.php

```diff
@@ -223,7 +223,7 @@ public function render(OutputFormat $oOutputFormat)
         $l = localeconv();
         $sPoint = preg_quote($l['decimal_point'], '/');
         $sSize = preg_match("/[\d\.]+e[+-]?\d+/i", (string)$this->fSize)
-            ? preg_replace("/$sPoint?0+$/", "", sprintf("%f", $this->fSize)) : $this->fSize;
+            ? preg_replace("/$sPoint?0+$/", "", sprintf("%f", $this->fSize)) : (string)$this->fSize;
```

Comment:
I presume this branch doesn't include the coding style change requiring space after cast operator.

Reply:
Yes, correct.